test/tasty/test/scala/tools/tastytest/TastyTestJUnit.scala

```diff
@@ -102,7 +120,8 @@ object TastyTestJUnit {
 
   final implicit class TryOps(val op: Try[Unit]) extends AnyVal {
     def eval: Unit = op match {
-      case Failure(err) => fail(err.toString)
+      case Failure(err) => err.printStackTrace()
```

Comment:
was this debugging output intentionally left in place?

Reply:
I should probably revert that